torch/library.py

```diff
@@ -91,10 +91,12 @@ def __del__(self):
 def impl(lib, name, dispatch_key=""):
     def wrap(f):
```

Comment:
Would it make sense to use `@functools.wraps` to preserve the function info?

Reply:
ye

Reply:
Ah that’s right